src/rpc/core_rpc_server.cpp

```diff
@@ -3171,7 +3171,7 @@ namespace cryptonote
           return false;
         }
 
-        res.distributions.push_back({std::move(*data), amount, "", req.binary, req.compress});
+        res.distributions.push_back({std::move(*data), amount, req.binary, true});
```

Comment:
I think a check for `req.binary && !req.compress` should be done, and an error returned in that case. This would only trigger on a (extremely unlikely) custom client.

Reply:
Wouldn't wallet clients before https://github.com/monero-project/monero/pull/7969/commits/267dc5a08ba4e362cb22f4b0aabde92eea92339a#diff-04cf14f64d2023c7f9cd7bd8e51dcb32ed400443c6a67535cb0105cfa2b62c3cR8283 trigger this?